TSPL.docc/ReferenceManual/Declarations.md

```diff
@@ -3876,6 +3876,18 @@ Access control is discussed in detail in <doc:AccessControl>.
   Declarations marked with the `public` access-level modifier can also be accessed (but not subclassed)
   by code in a module that imports the module that contains that declaration.
 
+- term `package`:
+  Apply this modifier to a declaration
+  to indicate that the declaration can be accessed
+  only by code in the same package as the declaration.
+  A package is a unit of code distribution
+  that you define in the build system you're using.
+  When the build system compiles code,
+  it specifies the package name
+  by passing the `-package-name` flag to the Swift compiler.
+  Two modules are part of the same package
+  if the build system specifies the same package name when building them both.
```

Comment:
The package name is set automatically in a typical Swift package but not for an Xcode project. Would it be appropriate to mention the Xcode build setting to set the package name here? For the record, it's `Package Access Identifier`.

Reply:
I added Xcode as another example next to the SwiftPM example in 8aafa213.  How's that look?

Reply:
That's good to know, I can understand that separation. The change looks fine to me.